### ISSUE TITLE
The android readme is really confusing

### ISSUE THREAD

**Issue by kallehed** [libsdl-org/SDL#7533]
I just spent a couple days porting my game to Android, and specifically this part of the README-android.md confused me incredibly:

> For more complex projects, follow these instructions:
>
> 1. Copy the android-project directory wherever you want to keep your projects
>   and rename it to the name of your project.
> 2. Move or symlink this SDL directory into the "<project>/app/jni" directory
> 3. Edit "<project>/app/jni/src/Android.mk" to include your source files

First it mentions an "android-project directory" which I thought was my OWN project. I think it would be more clear to say '1. Copy the directory "android-project" directory at "SDL/android-project" wherever you want to keep your project.'
Also, maybe the README could mention the purpose of the director? As I now understand it; the directory contains the basic glue code between Android's Java frontend, and the C/C++ SDL 'backend' we as developers write. It would have been useful to know that.

The second confusing part was 
>2. Move or symlink this SDL directory into the "<project>/app/jni" directory

Basically, I didn't know what a 'symlink' was. Maybe the word could be a hyperlink to the wikipedia page for symlink? Like: [Symlink](https://en.wikipedia.org/wiki/Symbolic_link)
Also... ehhh in the markdown code for that line, there is the word "project" before the "/app/jni" part, between two less than/greater than signs. But it seems markdown removes that part of the text?? So what you see is only "/app/jni" WHICH MAKES IT EVEN MORE CONFUSING

It's possible I'm stupid, but I would say this README could be greatly improved to make a bunch of peoples lives easier. Banged my head against this readme for multiple days, which wasn't really much fun.
So say if I should make a pull request or something.

**Comment by AntTheAlchemist** [comment]
That part always tripped me up as well, so it's not just you. I think symlink is a linux thing? A lot of the devs assume we should all have a linux background, which is the problem. 

The android-project is a very good template to start from, so maybe it needs to be clarified that it's a template project.

I ignore the symlink part and just insert a line into project-level Android.mk to point to where my SDL source code is, so it looks like this:
```
include $(call all-subdir-makefiles)
include ../../SDL/Android.mk
```

**Comment by AntTheAlchemist** [comment]
^ my SDL directory sits side-by-side with my project directories, so back 2 directories and into the SDL source code is what's needed for the relative path.

**Comment by kallehed** [comment]
Yeah exactly. It would be cool if the readme could mention this method as well

**Comment by AntTheAlchemist** [comment]
> Yeah exactly. It would be cool if the readme could mention this method as well

Go ahead and edit the readme to make it easier to read and add alternative to a symlink, it will be merged. Or paste the revised edit here. A lot of people have struggled with this, so it'll be appreciated.

**Comment by dustdfg** [comment]
Did you tried to run `androidbuild.sh` as mentioned in that README? It takes all work for you. In general, symlink is something like shortcut in windows

**Comment by AntTheAlchemist** [comment]
> Did you tried to run `androidbuild.sh` as mentioned in that README?

How do you run a .sh file? My Windows 10 PC doesn't recognise it.

**Comment by dustdfg** [comment]
>How do you run a .sh file? My Windows 10 PC doesn't recognise it.

It is not for windows

**Comment by AntTheAlchemist** [comment]
> It is not for windows

And that is the OP's problem. Not everyone uses Linux.

**Comment by kallehed** [comment]
> Did you tried to run `androidbuild.sh` as mentioned in that README? It takes all work for you. In general, symlink is something like shortcut in windows

Yes I did try androidbuild.sh at the time I started, but I think it failed for some unclear reason? Anyway, I currently understand basically everything now, no need to help me anymore 😅. This issue is mainly about me wanting to fix the documentation to be more clear.

**Comment by slouken** [comment]
Yes, feel free to create a PR with the documentation cleaned up to make more sense for you.

Thanks!

**Comment by kallehed** [comment]
Yes, made a pull request on my changes

**Comment by Starbuck5** [comment]
You can run shell scripts in Windows with git bash. Which almost everyone has installed, since you need it for Git on Windows.

* Not sure if that will work in this case, just sharing.

**Comment by slouken** [comment]
We merged your documentation improvements, thanks!
